### DISCUSSION TITLE
Documentation of shortcuts that are usable in the embedded diff editor

### DISCUSSION THREAD

**Discussion by alcroito** [jj-vcs/jj#2951] [Q&A] [answered]
Hi,

Is there a man page / help page / documentation for how to use the embedded diff editor that is started by `jj resolve`?

I was not able to find any.

**Comment by ilyagr** [comment, marked as answer]
Currently, the way to find this out is to use the mouse to go through the menus.

We'd like to improve this, but nobody has gotten around to it. The project is called `scm-diff-editor`, it lives in https://github.com/arxanas/scm-record.

**Reply by alcroito** [reply]
Thanks, it didn't occur to me that i can use the mouse to click on the TUI menus. I was frantically trying various keys like `TAB`, which didn't work.

**Comment by arxanas** [comment]
Filed https://github.com/arxanas/scm-record/issues/25 to track it, if anyone wants to pick it up 😁.
